From 4ebcfb929b6be63bacdba34f6aae0f5b51432d2c Mon Sep 17 00:00:00 2001
From: Jiri Slaby <jslaby@suse.cz>
Date: Fri, 24 Sep 2010 22:07:51 +0200
Subject: [SCSI] aha152x: enable PCMCIA on 64bit
Patch-mainline: yes
References: bnc#630652
Git-commit: 4ebcfb929b6be63bacdba34f6aae0f5b51432d2c

It was disabled when x86_64 was introduced, but it is reported to be
working on 64bit by two different people, so let's enable it back
again.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
Cc: "Juergen E. Fischer" <fischer@norbit.de>
Signed-off-by: James Bottomley <James.Bottomley@suse.de>
---
 drivers/scsi/pcmcia/Kconfig |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/drivers/scsi/pcmcia/Kconfig b/drivers/scsi/pcmcia/Kconfig
index 53857c6..ecc855c 100644
--- a/drivers/scsi/pcmcia/Kconfig
+++ b/drivers/scsi/pcmcia/Kconfig
@@ -11,7 +11,6 @@ if SCSI_LOWLEVEL_PCMCIA && SCSI && PCMCIA && m
 
 config PCMCIA_AHA152X
 	tristate "Adaptec AHA152X PCMCIA support"
-	depends on !64BIT
 	select SCSI_SPI_ATTRS
 	help
 	  Say Y here if you intend to attach this type of PCMCIA SCSI host
-- 
1.7.3.1

